### DISCUSSION TITLE
Login in app

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#7433] [Help] [answered]
Hi,

is there any way to put the Microsoft login inside a screen?
Problem is that any time our users restart their pcs they're logged out. 
Pressing on the 3x3 grid just to log in is really annoying.  

Sincerely

**Comment by R2bEEaton** [comment, marked as answer]
Hi @kasgithub8,

There is currently no way to do what you are describing, nor do I think there would be just from a security and UX standpoint. I've also had problems with logging out, but not every time I restart my PC. Are their browsers saving Cookies and remembering passwords? Also if any users share the same account, one will log off the other each time.

You could theoretically make an embed that displays the login screen, or include a button on the page that redirects users to the login screen (instead of having to use the 3x3 grid button).

Sorry this wasn't more helpful.

**Reply by kasgithub8** [reply]
Yeah I just added a button to redirect them yesterday.
Thanks for the information tho!
